### DISCUSSION TITLE
Neue CC Funktion

### DISCUSSION THREAD

**Discussion by LoonyBeLuna** [Scarwolf/p0weruser#149] [Ideas]
Hallo Scarwolf,
kannst du bitte die neue CC Funktion auch für p0weruser verfügbar machen?

Danke :)

**Comment by twobiers** [comment]
Sollte bereits gehen, wurde in ba91800177f47438c510f8ee833795f96bb7fecb eingebaut und in 2.9.0 released. 

*Edit: habe einen Fehler gemacht, der in 2429631 behoben wird. In 2.9.1 sollte es gehen

**Reply by LoonyBeLuna** [reply]
Danke, jetzt ist es sichtbar :)
Keine Ahnung wie sich das Script aktualisiert hat, aber jetzt gehts :)
Danke!
